pkg/kubelet/kuberuntime/kuberuntime_image.go

```diff
@@ -125,18 +125,15 @@ func (m *kubeGenericRuntimeManager) RemoveImage(image kubecontainer.ImageSpec) e
 }
 
 // ImageStats returns the statistics of the image.
-// Notice that current logic doesn't really work for images which share layers (e.g. docker image),
-// this is a known issue, and we'll address this by getting imagefs stats directly from CRI.
-// TODO: Get imagefs stats directly from CRI.
 func (m *kubeGenericRuntimeManager) ImageStats() (*kubecontainer.ImageStats, error) {
-	allImages, err := m.imageService.ListImages(nil)
+	filesystemUsages, err := m.imageService.ImageFsInfo()
```

Comment:
Sorry for the late review.  This isn't what we want here.  ImageStats should return statistics about the size of container images on the node.  ImageFsInfo returns stats about the total amount used on the image filesystem, including container writable layers, for example.

Reply:
Thanks for the pointer I take a look and adjust it.